src/lbaf/Model/lbsObject.py

```diff
@@ -226,17 +226,23 @@ def get_received(self) -> dict:
     @qoi
     def get_received_volume(self) -> float:
         """Return volume of communications received by object."""
-        return sum(v for v in self.__communicator.get_received().values()) if self.__communicator else 0
+        v : float = 0.0
```

Comment:
Is it necessary to hint that this is a float, as it is initialized at `0.0` immediately?

Reply:
Yes, not necessary, just more explicit